clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -287,7 +292,15 @@ public static ApiKeys forId(int id) {
     }
 
     public static boolean hasId(int id) {
-        return id >= MIN_API_KEY && id <= MAX_API_KEY;
+        if (id < MIN_API_KEY) {
```

Comment:
minor: combine the two if statements with logical or || .

Reply:
I actually feel like it's easier to read as is.